tests/test_override.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+
+from jrnl.override import apply_overrides, _recursively_apply, _get_config_node
+
+
+@pytest.fixture()
+def minimal_config():
+    cfg = {
+        "colors": {"body": "red", "date": "green"},
+        "default": "/tmp/journal.jrnl",
+        "editor": "vim",
+        "journals": {"default": "/tmp/journals/journal.jrnl"},
+    }
+    yield cfg
```

Comment:
Could you return instead of yield here? I don't see any iterating on this.

Reply:
The intent of `yield` here is to return the result of the fixture setup. In pytest fixtures, teardown code follows the `yield` call. At least, that's how I interpreted [the doc page](https://docs.pytest.org/en/reorganize-docs/yieldfixture.html). 

That said, there isn't any teardown code here. So a `return` should be good enough.